backend/base-backend-package/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@bufferapp/base-backend-package",
+  "version": "0.0.7",
+  "description": "A base package for writing new backend packages.",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "files": [
+    "lib/**/*"
+  ],
+  "scripts": {
+    "build": "tsc",
+    "format": "prettier --write \"src/**/*.ts\"",
+    "lint": "eslint . --ext .ts",
+    "test": "jest",
+    "prepare": "npm run build",
+    "prepublishOnly": "npm test && npm run lint",
+    "preversion": "npm run lint",
+    "version": "npm run format && git add -A src",
+    "postversion": "git push . HEAD && git push --tags"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/bufferapp/buffer-standards.git"
+  },
+  "homepage": "https://github.com/bufferapp/buffer-standards/backend/base-backend-package",
+  "keywords": [
+    "buffer",
+    "package example"
+  ],
+  "author": "Buffer engineering",
```

Comment:
Any thoughts on the author? I remember Harisson used to put his own handle for instance. I feel like there will be several authors since the packages will evolve. Do we have an engineering handle?

Reply:
Good one - I don't think we have an Engineering handle. However `Buffer engineering` sounds good to me 😄

Reply:
This handle comes from the local npm config if it's set, so not sure if we can really default it (not sure if that's an issue either) :)